source/features/dim-bots.tsx

```diff
@@ -40,7 +40,7 @@ function init(): void {
 
 	// Delay collapsing, but only after they're collapsed on load #5158
 	requestAnimationFrame(() => {
-		document.documentElement.classList.add('rgh-dim-bots--after-hover');
+		select('#repo-content-pjax-container .js-navigation-container')!.classList.add('rgh-dim-bots--after-hover');
```

Comment:
Let PJAX replaces the container with class instead of us doing the chore.

Note only using `#repo-content-pjax-container` does not work since the PJAX container is not replaced sometimes (e.g. switching tabs).